typed-racket-test/succeed/prop-evt.rkt

```diff
@@ -0,0 +1,39 @@
+#lang typed/racket/base
+
+(define ch ((inst make-channel Number)))
+
+
+(struct aaa0 ((evt : (Channelof Number)))
+  #:property prop:evt (struct-field-index evt))
+
+(thread (lambda ()
+          (channel-put ch 10)))
+
+(ann (sync (aaa0 ch)) Number)
+
+
+(struct aaa1 ([evt : (Evtof Number)])
+  #:property prop:evt 0)
+
+(thread (lambda ()
+          (channel-put ch 10)))
+
+(ann (sync (aaa1 ch)) Number)
+
+(struct aaa2 ([evt : (Channelof Number)])
```

Comment:
tests for cases where 1. the designated field type is not an event type 2. the lambda returns a non-event